src/main/java/com/nicoislost/inputs/KeyBind.java

```diff
@@ -0,0 +1,67 @@
+package com.nicoislost.inputs;
+
```

Comment:
I just want to point out that this is just one possible implementation.  I think the enum has value in the use as it provides automatic collection of the individual entities (that is, the `values` method is useful for looping through the available entities).

It also allows you to create common functionality which is easily accessible to for each case without a lot of additional code